src/js/_enqueues/wp/media/models.js

```diff
@@ -239,6 +239,6 @@ media.query = function( props ) {
 };
 
 // Clean up. Prevents mobile browsers caching.
-$(window).on('unload', function(){
+$(window).on('pagehide', function(){
```

Comment:
Origin is https://github.com/WordPress/wordpress-develop/commit/d2d01ad797e7894b6ea6d367c973f0c3d670003b: "Empty wp on unload to work around caching in iOS Safari."

This commit is 11 years old. Most likely this is no longer an issue in Safari. If so, we can most likely delete this entire event handler.